pkg/response/README.md

```diff
@@ -0,0 +1,34 @@
+# Event Logger
+
+This package enables response with event logging in go-restful apps.
+For the event logging please read logger/event package
+
+## Usage
+
+### Importing
+
+```go
+import "github.com/AccelByte/go-restful-plugins/v3/pkg/response"
+```
+
+### Write Response Success
+
+```go
+Write(request, response, httpStatusCode, serviceType, eventID, message, entity)
+```
+
+### Write Response Error
```

Comment:
Add that we recommend to use "github.com/pkg/errors" to create error and wrap the errors with stack trace before being passed here, because it will log the stack trace too to help with debugging

Reply:
okay, added

Reply:
uh, where?

Reply:
> uh, where?

line 32
`We recommend use `"github.com/pkg/errors"` to create error and wrap the errors with stack trace to help with debugging`

Reply:
ah OK